.github/workflows/run_helm_tests.yaml

```diff
@@ -0,0 +1,151 @@
+name: Helm tests
+
+concurrency:
+  group: helm-${{ github.event_name }}-${{ github.ref }}
```

Comment:
Q: should we use `github.ref` or `github.sha` as the last segment?  I saw another workflow that used `sha`.  :thinking: 

Perhaps using `ref` is preferred, since it'll abort ongoing runs for the same branch.